Common/cpp/NativeModules/NativeReanimatedModule.h

```diff
@@ -130,6 +131,11 @@ class NativeReanimatedModule : public NativeReanimatedModuleSpec {
   jsi::Value measure(jsi::Runtime &rt, const jsi::Value &shadowNodeValue);
 
   void initializeFabric(const std::shared_ptr<UIManager> &uiManager);
+
+  std::string obtainPropFromShadowNode(
```

Comment:
Changing `std::string` into `jsi::Value` everywhere is a big change, since jsi::Value is a temporary object and cannot be always accessed from other threads and it would require a lot of testing. Can I do this in separate PR?